src/components/Account.vue

```diff
@@ -38,55 +50,145 @@
+                            </path>
+                        </clipPath>
+                    </defs>
+                </svg>
+            `);
+            /* tslint:enable max-line-length */
+        }
     }
 </script>
 
 <style scoped>
     .account {
-        width: 100%;
         display: flex;
-        flex-direction: row;
         align-items: center;
         justify-content: space-between;
         padding: 1.75rem 2rem;
         box-sizing: border-box;
         flex-shrink: 0;
         font-size: 2rem;
-        line-height: 1.25;
+        line-height: 1.2;
+    }
+
+    .account.row {
+        width: 100%;
+        flex-direction: row;
+    }
+
+    .account.column {
+        flex-direction: column;
     }
 
     .identicon-and-label {
         display: flex;
-        flex-direction: row;
         align-items: center;
+    }
+
+    .row .identicon-and-label {
+        flex-direction: row;
         overflow: hidden;
         min-width: 5.625rem;
     }
 
+    .column .identicon-and-label {
+        flex-direction: column;
+    }
+
     .identicon {
+        flex-shrink: 0;
+        position: relative;
+    }
+
+    .row .identicon {
         width: 5.625rem;
         height: 5.625rem;
-        flex-shrink: 0;
         margin-right: 1.5rem;
     }
 
-    .label {
-        white-space: nowrap;
-        font-weight: 600;
-        overflow: hidden;
+    .column .identicon {
+        width: 9rem;
+        height: 9rem;
+        margin-bottom: 1.5rem;
     }
 
-    .label div {
-        opacity: 0.7;
+    .identicon .account-image {
+        width: 100%;
+        height: 100%;
+        clip-path: url(#nimiq-hexagon-clip);
+    }
+
+    .identicon .outline {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        background-repeat: no-repeat;
+        background-position: 0 center;
+        background-size: contain;
+        background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" fill="none" width="146" height="146"><path d="M113.8 16.2l28.7 49.6c2.5 4.5 2.5 10 0 14.4l-28.7 49.6a14.4 14.4 0 0 1-12.5 7.2H44.1c-5.2 0-10-2.7-12.5-7.2L2.9 80.2C.4 75.7.4 70.2 3 65.8l28.7-49.6C34.2 11.7 38.9 9 44 9h57.2c5.2 0 10 2.7 12.5 7.2z" clip-rule="evenodd" stroke="%231f2348" stroke-width="2" opacity=".2"/></svg>');
+    }
+
+    .label,
+    .wallet-label {
         overflow: hidden;
         text-overflow: ellipsis;
+    }
+
+    .wallet-label {
+        margin-bottom: 0;
+    }
+
+    .row .label:not(.editable) {
+        opacity: 0.7;
         padding-left: 1rem;
     }
 
-    .balance {
+    .row .label,
+    .row .wallet-label {
+        white-space: nowrap;
+        font-weight: 600;
+    }
+
+    .column .label,
+    .column .wallet-label {
+        max-width: 16.5rem;
```

Comment:
I am trying the code in storybook, and this seems like an unnecessary limitation. Why can't the label input and wallet label not be wider, when the space allows it? Something like `80%` is better, I think.

Reply:
I wanted to give it a size that looks good with the fixed identicon size.
I think if the label would be much wider than the identicon, it wouldn't look good.
Also with a fixed size, it's ensure that the component looks consistent everytime it's used.

Would you say removing the limitation wins over the consistent style?

Reply:
But the label _can_ be much longer than the identicon is wide. Also, see the designs, where the input is always wider than the identicon:
![image](https://user-images.githubusercontent.com/1828163/56024392-39144380-5d10-11e9-9158-1a92af9f59af.png)
![image](https://user-images.githubusercontent.com/1828163/56024435-4d584080-5d10-11e9-8993-3721c54db4f7.png)
etc.

Reply:
The default max width is already wider than the identicon.
The screens you posted should in my opinion be based on the `AccountDetails` component which overwrites all sizes anyways.

Reply:
Ah, you are right. I overlooked that component. Ok, good to go!